Code/Editor/Plugins/EditorAssetImporter/AssetImporterWindow.cpp

```diff
@@ -673,22 +678,25 @@ void AssetImporterWindow::HandleAssetLoadingCompleted()
     m_qtFileWatcher.addPath(m_assetImporterDocument->GetScene()->GetManifestFilename().c_str());
 }
 
-void AssetImporterWindow::FileChanged(QString path)
+void AssetImporterWindow::ReloadCurrentScene(bool warnUser)
 {
     if (m_isSaving)
     {
         return;
     }
 
-    QString promptMessage([this]()
-    {
-        if(m_rootDisplay->HasUnsavedChanges()) 
+    QString promptMessage(
+        [this]()
         {
-            return tr("The file %1 has been changed outside of the scene settings tool. This tool will be reloaded and any unsaved changes will be lost. \n\n"
-                        "To prevent this from occuring in the future, do not modify the scene file or scene manifest outside of this tool while this tool has unsaved work.");
-        }
-        return  tr("The file %1 has been changed outside of the scene settings tool. This tool will be reloaded.");
-    }());
+            if (m_rootDisplay->HasUnsavedChanges())
```

Comment:
Not for this PR but do we want to merge the unsaved changes after the file is modified outside this tool (i.e. reload the file and apply the unsaved changes)? We may throw an error only if we couldn't resolve it automatically.

Reply:
Good idea, I've made an internal ticket for that.